test/faker-lib.js

```diff
@@ -0,0 +1,12 @@
+const faker = require('faker');
+module.exports = { 
+    quiz_name : faker.name.firstName(),
+    description : faker.commerce.productDescription(),
+    prizeMoney : parseInt(faker.commerce.price()),
+    starttime : parseInt(faker.date.future().getTime()/1000),
+    time: parseInt(faker.date.past().getTime()/1000),
+    duration : 100,
+    questionHash : faker.git.commitSha(),
```

Comment:
Create questions and then get the hash of those questions.
You can refer https://github.com/biswajitpatra/Blockchain-Quiz/blob/af1ebda55177d5e4c8450673d5bfbef8f651241c/client/components/OrganizerUploadForm.js#L67-L98  for encoding and hashing the questions